infra/aws/aws-costexplorer-export/README.org

```diff
@@ -0,0 +1,85 @@
+#+TITLE: AWS CostExplorer Export
```

Comment:
@BobyMCbobs Is it possible to use the Markdown format for all the docs ? Let's use a format well adopted by the community.

Reply:
Swapped out the format in 0d224a1